<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bilge-Katalog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="CatList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QListWidget" name="ExpList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actNewCat"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actNewDir"/>
     <addaction name="actNewFile"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actDel"/>
    <addaction name="separator"/>
    <addaction name="actCut"/>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
    <addaction name="separator"/>
    <addaction name="actInfo"/>
    <addaction name="separator"/>
    <addaction name="actSet"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actShowExpBar"/>
    <addaction name="actShowEditBar"/>
    <addaction name="actShowSearchBar"/>
    <addaction name="actShowStatBar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="StatBar"/>
  <widget class="QToolBar" name="ExpBar">
   <property name="windowTitle">
    <string>Exploring Bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="EditBar">
   <property name="windowTitle">
    <string>Editing Bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="SearchBar">
   <property name="windowTitle">
    <string>Searching Bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actNewCat">
   <property name="text">
    <string>&amp;Create Catalog</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actNewDir">
   <property name="text">
    <string>&amp;Directory</string>
   </property>
  </action>
  <action name="actNewFile">
   <property name="text">
    <string>F&amp;ile</string>
   </property>
  </action>
  <action name="actDel">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actCut">
   <property name="text">
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="text">
    <string>Cop&amp;y</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actInfo">
   <property name="text">
    <string>&amp;Info</string>
   </property>
  </action>
  <action name="actSet">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actShowExpBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show e&amp;xploring bar</string>
   </property>
  </action>
  <action name="actShowEditBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;editing bar</string>
   </property>
  </action>
  <action name="actShowSearchBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;searching bar</string>
   </property>
  </action>
  <action name="actShowStatBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show s&amp;tatus bar</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actBack">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actForward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actUp">
   <property name="text">
    <string>Up</string>
   </property>
  </action>
  <action name="actRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actShowEditBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actShowExpBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>ExpBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actShowSearchBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>SearchBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actShowStatBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>StatBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
